calico_kubernetes/calico_kubernetes_cni.py

```diff
@@ -60,19 +81,23 @@ def create(args):
     :param args: dict of values to pass to other functions (see: validate_args)
     """
     container_id = args['container_id']
+    namespace = args['namespace']
+    pod_name = args['pod_name']
     netns = args['netns']
     interface = args['interface']
     net_name = args['name']
-    subnet = args['subnet']
+    profile_name = '%s_%s_%s' % (namespace, pod_name, container_id[:12])
```

Comment:
Hmm, we don't want one profile per pod do we? It doesn't look like we have advanced policy in this plugin yet, is that right? And in advanced policy mode, how were we handling profiles?